torch/nn/modules/batchnorm.py

```diff
@@ -10,6 +10,7 @@
 from .lazy import LazyModuleMixin
 from .module import Module
 
+from torch._C._distributed_c10d import ProcessGroup
```

Comment:
Hm maybe this is why the typing was `Optional[Any]` before. Won't this be a problem if PyTorch is built with `USE_DISTRIBUTED=0` ?

Reply:
Follow-up question: why is `SyncBatchNorm` not in distributed :p

Reply:
Don't know...

Reply:
Shall we refactor that into distributed? Or what is the best way to go about it?

Reply:
I think refactoring it would be quite a bit of work.
I think that for this PR you can move back to `optional[Any]`. Sorry for the back and forth.

Reply:
But still do the is not None comparison?

Reply:
Yes, that is right.

Reply:
OK, I undid the changes. Next thing on the list is how to deal with the CLA. My company has signed it so I cannot sign it individually, what do I need to do?